.linthtmlrc.json

```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
+  "attr-bans": [
+    "align",
+    "background",
```

Comment:
Please do not add config files to PR next time

Reply:
it was the last time
i promise :)